components/docs-chef-io/content/habitat/plan_settings.md

```diff
@@ -101,7 +101,7 @@ pkg_version
 {{< /foundation_tabs_panels >}}
 
 {{< note >}}
-Habitat uses the guidelines of the semantic versioning specification to determine version order (see https://semver.org/). Keep in mind that including a hyphen in your version indicates that the package is a pre-release that will be given lower precedence to other versions.
+Habitat uses the guidelines of the semantic versioning specification to determine version order (see https://semver.org/ ). Keep in mind that including a hyphen in your version indicates that the package is a pre-release that will be given lower precedence to other versions.
```

Comment:
This will work better:
```suggestion
Habitat uses the guidelines of the semantic versioning specification to determine version order (see <https://semver.org/>). Keep in mind that including a hyphen in your version indicates that the package is a pre-release that will be given lower precedence to other versions.
```